Src/FluentAssertions/Execution/AssertionScope.cs

```diff
@@ -84,6 +84,11 @@ public AssertionScope(string context)
         /// </summary>
         public string Context { get; set; }
 
+        /// <summary>
+        /// Gets the number of assertions in the assertion scope.
```

Comment:
I think it would be better to describe it as "Gets the number of assertions that were executed since the AssertionScope was created and the last time DiscardAny was called.

Reply:
Good you mention it, I was thinking about how this would behave if a chain of assertions fails before executing all assertions in the chain.